### PR TITLE
[17.0][MIG] sale_order_report_product_image: Migration to 17.0

### PR DESCRIPTION
**Pull request by JoseIbarraHdz**
- update version to 17.0.1.0.0 

image

### CODE REVIEW

**Comment by edy1192** [comment]
Functional test LGTM 👍🏼

**Comment by edy1192** [comment]
Hi @manuelregidor could you check this, please?

cc @JoseIbarraHdz

**Comment by manuelregidor** [comment]
@edy1192 You need to have two commits: one for the pre-commit auto fixes and another one for the migration itself.

**Review by antoniocanovas — APPROVED** [review]
Functional review OK !!

**Comment by JoseIbarraHdz** [comment]
Hi @manuelregidor could you check this please? There are the two commits you mentioned above.

Regards.

cc. @edy1192

**Review by manuelregidor — APPROVED** [review]
Technical review. LGTM

File: pandoc-3.4-1-amd64.deb
Comment (pedrobaeza) [inline pandoc-3.4-1-amd64.deb:1]: This file shouldn't be present.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
There's a mix of changes in commits: some of them should be on the migration commit (like the README ones), and some on the pre-commit auto-fixes.

**Comment by JoseIbarraHdz** [comment]
Hi @pedrobaeza I did the changes you requested, can you please check?

**Review by pedrobaeza — APPROVED** [review]
OK, there are still some mix of changes between commits, but for this time is enough. Please follow all the steps of the migration guide the next time for avoiding this kind of things. 

/ocabot merge nobump

**Comment by JoseIbarraHdz** [comment]
Hi @pedrobaeza thanks for the approval,  I'll follow the guide steps next time, by the way, it is already merged?

Regards.

**Pull request merged**